README.md

```diff
@@ -8,6 +8,41 @@ Yet another note taking tool.
 cargo tauri dev
 ```
 
+## Setup environment
+This guide focuses on how to setup your development system based on ubuntu:jammy.
```

Comment:
Since we produce an appimage and appimage build is bound to the oldest supported platform*, ubuntu:20.04 (focal) should be the reference platform. I also prefer the actual version number instead of the name. Please change.

```suggestion
This guide focuses on how to setup your development system based on ubuntu:20.04.
```

_(*this is due to libc dependencies, see [Binaries compiled on old enough base system](https://github.com/AppImage/AppImageKit/wiki/Creating-AppImages/cc2441518975caca23e9ce2dba6f08a22c678d1e))_

Reply:
With commit c337707716c614b8271430b3999e47c3e33cc656 the version number is used and the information of the use of ubuntu is now moved to the summary of the collapsible section. Is the use of Ubuntu 20.04 a __must__ or is it possible to build and run the project on other OS? I think by referencing Ubuntu 20.04 in the collapsible section header relatives Ubuntu 20.04  as the OS for development.

Reply:
It is not a must, but recommended due to appimage requirements. When you don't need to support Ubuntu 20.04, you can use a newer version and / or a different distribution.